.vib/spring-cloud-dataflow/cypress/cypress/integration/utils.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+export let random = (Math.random() + 1).toString(36).substring(7);
+
```

Comment:
These are added as separate methods since I am using them to achieve the desired application state without creating dependencies between tests.

Reply:
I'm not sure these functions can be considered utility ones instead of proper tests. It's true you need them to achieve the desired state for your tests, but I think the issue here is their required starting state. 

For example, the `check created task and schedule one` test could be `create task and schedule one` and we will avoid having the `createATask` function. Moreover, here you are in fact testing that the imports function properly and that the task creation process works.

As I understand it, utility functions should be small functions that help in the creation of the tests such as `random` or `obtainCredentials` functions.

Reply:
I also think these methods don't belong to the `utils.js`.
As agreed on our call, I'll create a new .js file with these methods which are used for application state preparation.